packages/beeq/src/components/select/_storybook/bq-select.stories.tsx

```diff
@@ -257,6 +276,16 @@ export const Multiple: Story = {
   },
 };
 
+export const MultipleCustomRender: Story = {
+  render: Template,
+  args: {
+    'keep-open-on-select': true,
+    multiple: true,
+    customTags: true,
+    value: ['running', 'biking', 'pizza'],
+  },
+};
```

Comment:
For custom tags rendering we should also handle properly the clear action.

https://github.com/Endava/BEEQ/assets/328492/631b0bb0-02ec-417a-b2bc-a2125679eea2

Reply:
Done